package.json

```diff
@@ -21,7 +21,7 @@
     "@bcrs-shared-components/approval-type": "1.1.3",
     "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.15",
-    "@bcrs-shared-components/business-lookup": "1.3.4",
+    "@bcrs-shared-components/business-lookup": "1.3.15",
```

Comment:
The intermediate versions are probably just dependency updates (like enums). Thanks for checking!